File: documentation-examples-scala/src/scala/dev/typr/foundations/docs/postgresql/UuidType.scala
package dev.typr.foundations.docs.postgresql
import dev.typr.scalafoundations.*
import dev.typr.scalafoundations.data.*


import java.util.UUID

@SuppressWarnings(Array("unused"))
object UuidType:
  //start
  val uuidType: PgType[UUID] = PgTypes.uuid
  //stop

File: documentation-examples-java/src/java/dev/typr/foundations/docs/postgresql/UuidType.java
package dev.typr.foundations.docs.postgresql;

import dev.typr.foundations.PgType;
import dev.typr.foundations.PgTypes;
import java.util.UUID;

@SuppressWarnings("unused")
public class UuidType {
  // start
  PgType<UUID> uuidType = PgTypes.uuid;
  // stop
}
